lib/formatters/__tests__/verboseFormatter.test.js

```diff
@@ -18,12 +17,13 @@ describe('verboseFormatter', () => {
 
 		const output = prepareFormatterOutput(results, verboseFormatter);
 
-		expect(output).toBe(stripIndent`
-      1 source checked
-       path/to/file.css
+		expect(output).toBe(
+			`
+1 source checked
+ path/to/file.css
 
-      0 problems found
-    `);
+0 problems found`.trimStart(),
```

Comment:
[note] Reluctantly, I use `trimStart()` only in this file in order to trim the newline at the beginning of the string.